cmd/celestia-appd/cmd/download_genesis.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// chainIDToSha256 is a map of chainID to the SHA-256 hash of the genesis file for that chain ID.
+// To add a new chain-id, download the genesis file from the networks repo and compute the SHA-256 hash.
```

Comment:
[nit] network's instead of networks

Reply:
network's is the possessive form of networks and I don't think that applies here